internal/command/operator.go

```diff
@@ -82,7 +113,7 @@ func operatorDeploy() *cobra.Command {
 	flags := cmd.PersistentFlags()
 	flags.StringVar(&version, "version", "", "Specifies a specific version of the operator to install, defaults to latest")
 	flags.StringVar(&registry, "registry", defaultRegistry, "Specifies an alternative image registry to use for the operator image")
-	flags.StringVar(&credentials, "credentials", "", "Specifies the location of the credentials file to use for docker image pull secrets")
+	flags.StringVar(&registryCredentialsPath, "registry-credentials-path", "", "Specifies the location of the credentials file to use for docker image pull secrets")
```

Comment:
Suggestion: write in the help output what it does if the path is not provided (gets the creds from context and errors out if not provided)?

Reply:
I think that the adjustments to the help text in a7417f7edb7176ab372c6eb18d71be1f67c29294 will help explain this.